spec/plans/provision.fmf

```diff
@@ -75,3 +75,39 @@ link:
       - implemented-by: /tmt/steps/__init__.py
       - implemented-by: /tmt/queue.py
       - verified-by: /tests/multihost
+
+/ansible:
+    summary: Per-guest Ansible customization
+
+    story:
+        As a user I want to assign an Ansible group and host variables
+        for a provisioned guest to be reflected in the generated inventory.
+
+    description: |
+        Each provisioned guest may define an optional ``ansible`` key
+        to define its corresponding Ansible inventory section.
+
+        - ``group``: Assigns the guest to an Ansible specific group. If omitted,
+          the ``role`` key would serve as the Ansible group name.
+          If ``role`` is not set either, the host will be placed in the
+          'ungrouped' group.
```

Comment:
Please, include link to https://tmt.readthedocs.io/en/stable/plugins/provision.html#common-keys` so users can transition from here to the documentation of `role` key (plus, `ungrouped` is marked as a literal name, but we should use backticks for such highlight):

```
the :ref:`role </plugins/provision/common-keys>` key would serve as the Ansible group name. If ``role`` is not set either, the host will be placed in the ``ungrouped`` group.
```